cordova-js-src/splashscreen.js

```diff
@@ -148,29 +152,27 @@ function init(config, manifest) {
     extendedSplashProgress.classList.add('win-medium');
     extendedSplashProgress.classList.add('win-ring');
 
+    extendedSplashImage.style.maxWidth = "100%";
```

Comment:
Could you please explain, what's the point in having `maxWidth` and `maxHeight`  defined twice - here and in css file?